peerspective-python-developer.html

```diff
@@ -0,0 +1,43 @@
+---
+title: Python developer
+company: Peerspective
+location: Richmond, London
+contract: permanent
+contact:
+    name: Andrew Thomas
+    email: andrew@peerspective.io
+created: !!timestamp 2018-04-25
+tags:
+  - python
+  - flask
+  - SQLAlchemy
+  - Celery
+  - Go
+  - Redis
+  - DynamoDB
+  - MySQL
+  - S3
+---
+
+# Backend or Full Stack
+Looking for backend and full stack python developers
+
+We are working with one of the fastest growing startups in the UK to find a Full Stack Python developer or a purely Backend Python developer
```

Comment:
You might want to say a bit more about what your company does, what the work culture is like and what kinds of people you are looking at.